internal/service/deployment_manager_handler.go

```diff
@@ -579,8 +579,20 @@ func (h *DeploymentManagerHandler) fetchProfile(ctx context.Context,
 		return
 	}
 
-	// Fetch the kubeconfig that was used to register the hub, and then use it to fetch the
-	// admin kubeconfig of the hub:
+	// First we will try to get the admin Kubeconfig from the namespace of the cluster. This
+	// will work if the cluster was provisioned using ACM and assisted installer.
+	assistedInstallerAdminKubeconfig, err := h.fetchAssistedInstallerAdminKubeconfig(
+		ctx,
+		h.hubClient,
+		cluster,
+	)
+	if err != nil {
+		return
+	}
+	if assistedInstallerAdminKubeconfig != nil {
+		result, err = h.makeProfile(assistedInstallerAdminKubeconfig)
```

Comment:
Maybe worth having some log here?
I.e. a debug/info log that indicates which kubeconfig is used.

Reply:
Makes sense, will do it.